packages/bot-web-ui/src/stores/toolbar-store.js

```diff
@@ -84,7 +84,7 @@ export default class ToolbarStore {
     }
 
     @action.bound
-    onResetCancelButtonClick() {
+    toggleResetModal() {
```

Comment:
It would be better for `toggleResetModal` to be renamed to `toggleResetDialog` or `toggleDialog` since you're using Dialog now.